feat(token-purchase): add GET sell flow and swap direction toggle

Matinj031 opened the pull request:
### This PR Provides:

- Adds a fully functional Sell flow that mirrors the existing Buy flow for GET.
- Introduces a horizontal swap toggle to switch trade direction between Buy and Sell without clearing user input.
- In Sell mode, swaps from GET to the selected counterparty token `(SOL, USDC, USDT)`; in Buy mode, swaps from the selected counterparty token to GET.
- Success message now shows a single Solana Explorer transaction link with cluster detection.

Matinj031 committed:
feat(token): add explorer links on swap success

Show transaction links (Solscan and Solana Explorer) in the success
message after a successful swap, enabling quick verification of the
transaction.

- Store lastSignature and build explorer URLs
- Detect network cluster from RPC URL to generate correct links
- Extend success message visibility to 20s and clear lastSignature
- Extend error message visibility from 5s to 10s
- Add styles for transaction links

Matinj031 committed:
refactor(token): remove solscan tx link, use Solana Explorer only

Matinj031 committed:
feat(token): add sell mode with accessible trade toggle

Introduce sell mode to TokenPurchase with a buy/sell toggle and improved
UX/accessibility:
- Add tradeMode (buy/sell), keyboard-accessible toggle, and dynamic
  primary button label
- Implement red sell button styling and update amount info text
- Enhance canSwap logic and add disabledReason messaging (e.g.
  insufficient GET, fetching quote, no route)
- Support sell quotes (GET -> pay token) and refine buy quotes with
  buffered equivalent cost
- Update handleSwap to respect trade direction, add targeted balance
  checks (SOL fees on buy, GET on sell), and improve error handling

sanaderi commented on components/token/TokenPurchase.vue:
```diff
@@ -19,15 +19,32 @@
+        @keydown.enter.prevent="toggleTradeMode"
+        @keydown.space.prevent="toggleTradeMode"
+      >
         <svg
```
use `<v-icon>md:swap_horiz</v-icon>` instead svg

sanaderi commented on components/token/TokenPurchase.vue:
```diff
@@ -80,10 +97,10 @@
     <button
       :disabled="!canSwap"
-      :class="['figma-green-buy-btn', { loading: swapping }]"
+      :class="[tradeMode === 'sell' ? 'figma-red-sell-btn' : 'figma-green-buy-btn', { loading: swapping }]"
```
plz use [vuetify color palette ](https://vuetifyjs.com/en/styles/colors/#material-colors) instead use custom class.

sanaderi commented on components/token/TokenPurchase.vue:
```diff
@@ -80,10 +97,10 @@
+      :class="[tradeMode === 'sell' ? 'figma-red-sell-btn' : 'figma-green-buy-btn', { loading: swapping }]"
       @click="handleSwap"
     >
-      {{ swapping ? 'Processing...' : 'Buy GET' }}
```
Please use `$GET` instead of `GET`, because `GET` has a wide range of meanings.

sanaderi commented on components/token/TokenPurchase.vue:
```diff
@@ -113,12 +130,31 @@
     </div>
 
+    <!-- Disabled reason (shown only when button is disabled and no explicit errorMessage is set) -->
+    <div
```
Plz use `v-alert` component instead use custom style 
https://vuetifyjs.com/en/components/alerts/

sanaderi commented on components/token/TokenPurchase.vue:
```diff
@@ -408,14 +466,51 @@ const canSwap = computed(() => {
+    }
+  }
+
+  if (isCalculating.value) return 'Fetching quote...'
```
The page vibrates while typing a value due to the 'Fetching data' message. Please fix this. The message is good and necessary,  you can display 'Fetching...' on the Swap (Buy/Sell) button.

Matinj031 committed:
Merge branch 'upgrade-v3' of https://github.com/GamaEdtech/front into feat/solana-wallet-integration

Matinj031 committed:
refactor(token): migrate TokenPurchase to Vuetify and $GET branding

- replace inline SVG with v-icon (swap_horiz)
- switch primary action to v-btn with success/error palette
- show errors/success via v-alert and remove disabledReason messaging
- add "Fetching..." state to primaryButtonLabel when quoting
- update copy to use $GET consistently, including balance errors
- tweak explorer link styling and modal copy
- add disabled v-btn style override for better UX
